.ci/esy-build-steps.yml

```diff
@@ -0,0 +1,12 @@
+# Cross-platform set of build steps for building esy projects
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '8.9'
+  - script: npm install -g esy@0.6.2
+    displayName: 'npm install -g esy@0.6.2'
+  - script: esy install
+    displayName: 'esy install'
+  - script: esy build
+    displayName: 'esy build'
```

Comment:
Is it correct that these steps are running in `cmd`?

Reply:
It is correct that they are running in `cmd` - one of the main benefits of using `esy`, for me, is that I don't have to change how I work as a windows developer. 

The set of steps described in `esy-build-steps.yml` is the same for OSX, Linux, and Windows - it's really nice to have a consistent workflow across all platforms:
- Install `esy` (`npm install -g esy`)
- Install dependencies (`esy install`)
- Build (`esy build`)